crates/events/src/tracer.rs

```diff
@@ -227,6 +228,29 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_event_trace_emit_event() {
+        let event = {
+            let ev = Event::new(
+                EventTypeBuilder::new_with_type("test").build(),
+                EventLevel::Tx,
+            );
+            let mut events = Vec::with_capacity(1);
+            emit_event(ev, &mut events);
```

Comment:
...here instead

Reply:
it's right, no? I think it's more like this: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=d5cf2e4b13d8cd82d7d8046657a9243b